README.md

```diff
@@ -80,20 +80,24 @@ We will backport bugfixes--but not new features--into older versions of
 
 #### Compatibility matrix
 
-|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 |
-|---------------------|----------------|----------------|----------------|----------------|
-| client-go 1.4       | +              | ✓              | -              | -              |
-| client-go 1.5       | +              | +              | -              | -              |
-| client-go 2.0       | +              | +              | ✓              | -              |
-| client-go 3.0 beta  | +              | +              | +              | ✓              |
-| client-go HEAD      | +              | +              | +              | +              |
+|                     | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 | Kubernetes 1.7 |
+|---------------------|----------------|----------------|----------------|----------------|----------------|
+| client-go 1.4       | +              | ✓              | -              | -              | -              |
+| client-go 1.5       | +              | +              | -              | -              | -              |
+| client-go 2.0       | †              | †              | ✓              | -              | -              |
```

Comment:
I think there should always be a release that is a + between the check and the dagger marked releases, or we are violating a deprecation policy?